src/features/wallet/root/useActivityItem.tsx

```diff
@@ -31,8 +31,12 @@ import { Colors } from '../../../theme/theme'
 import { getMakerName } from '../../../utils/stakingClient'
 import { RootState } from '../../../store/rootReducer'
 
+// TODO import from helium-js once released
+class RewardsV2 extends RewardsV1 {}
```

Comment:
the helium-js PR basically does the same thing as this line since we're operating under the assumption that rewards v1 & v2 should contain the same data